bosh/bosh-release/jobs/kibosh/templates/start.erb

```diff
@@ -33,11 +36,27 @@ export CF_BROKER_NAME=<%= p("kibosh.cf.broker_name") %>
 export CF_SKIP_SSL_VALIDATION=<%= p("kibosh.cf.skip_ssl_validation") %>
 <% end %>
 
-mkdir -p "$LOG_DIR" "$RUN_DIR" "$HELM_CHART_DIR"
+if [ ! -d "$LOG_DIR" ] ; then
+  mkdir -p "$LOG_DIR"
+  chown -R vcap:vcap "$LOG_DIR"
```

Comment:
Protecting the `chown` such that it's only run if the dir is missing could introduce upgrade issues. Any newly introduced files put into place by the release from version A to B would not be `chown`ed because release version A would have already created the directory. Breaking out the `chown` is safer for upgrades (this applies to the other 3 below `chown` commands as well.

Reply:
That change was trying to attempt to fix this comment:
# If one of these directories is very large, chowning might take a very long time.
# Consider only chowning if dir did not yet exist in the previous step

I'll return it back to what it was before, which is safer.